python/src/pyripherals/peripherals/DDR3.py

```diff
@@ -641,6 +639,14 @@ def data_to_names(self, chan_data, old=False):
                or the timestamp steps are not all the same
         """
 
+        if bitfile_version is None:
+            # Old code, hasn't been updated to pass bitfile_version from .h5 file header
+            # Default to FPGA bitfile version, or version 1 if the FPGA bitfile version is None
+            if self.fpga.bitfile_version is None:
+                bitfile_version = 1
+            else:
+                bitfile_version = self.fpga.bitfile_version
+
```

Comment:
Later, if `bitfile_version` remained as `None` when passed in, we use the FPGA's `bitfile_version` if it has been initialized and has one, otherwise we default to version 1